src/web-ui/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@aws-amplify/ui-components": "^1.1.1",
-    "@aws-amplify/ui-react": "^1.0.6",
-    "aws-amplify": "^3.3.20",
-    "bootstrap": "^4.4.1",
-    "react": "^16.14.0",
-    "react-bootstrap": "1.0.0",
-    "react-dom": "^16.14.0",
-    "react-scripts": "^4.0.3",
-    "react-webcam": "^5.0.2",
+    "@aws-amplify/ui-components": "^1.9.6",
+    "@aws-amplify/ui-react": "^2.3.0",
+    "aws-amplify": "^4.3.14",
+    "bootstrap": "^4.6.1",
+    "react": "^17.0.2",
+    "react-bootstrap": "1.6.4",
+    "react-dom": "^17.0.2",
+    "react-scripts": "^5.0.1",
+    "react-webcam": "^7.0.1",
```

Comment:
I cannot find a changelog for react-webcam, have you checked manually to see if all still works as expected?